frontend/shadow-cljs.edn

```diff
@@ -67,6 +67,11 @@
     :output-wrapper false
     :warnings {:fn-deprecated false}}
 
+   :js-options
+   {:entry-keys ["module" "browser" "main"]
+    :resolve {"renderer" {:target :file
+                          :file "resources/public/js/renderer/renderer.js"}}}
```

Comment:
If the module is going to be imported on the code, it should not be located on resources/public/, resource public is a directory from where the browser (example: from index.html) to be imported.

But as you already import it from code inside src, on generating the bundle, it will be already be bundled inside public/js by shadow-cljs.

Reply:
You're right, this is a leftover from a previous version, shouldn't be here